### DISCUSSION TITLE
Doubts regarding Project Idea Smart Meeting Notes

### DISCUSSION THREAD

**Discussion by AlexFierro9** [openvinotoolkit/openvino#16325] [Google Summer of Code]
Can anyone please point tell how can I get in touch with the mentors @eaidova  and @andrei-kochin, I have some doubts regarding the project scope and would like to clear them before submitting my proposal.

**Comment by andrei-kochin** [comment]
@AlexFierro9 you are in the right place. Please post you queries

**Comment by andrei-kochin** [comment]
Seems there is [another discussion](https://github.com/openvinotoolkit/openvino/discussions/16385#discussion-4982646)
